projects/netgrif-components-core/src/assets/i18n/en.json

```diff
@@ -457,5 +463,16 @@
     },
     "dynamicNavigation": {
         "couldNotResolveView": "An error has occurred during the view resolution process. The view could not be displayed"
+    },
+    "impersonation": {
+        "user": {
+            "successfullyRepresented": "User successfully represented",
```

Comment:
why is it sometimes called "representation / represented" and othertimes "impersonation / impersonated"?

Reply:
good question, which do you think is better? Ill change the rest to ensure consistency